registry/vclab/star/container.yaml

```diff
@@ -0,0 +1,8 @@
+docker: tipsarin/star2.7.2a
+url: https://github.com/vclabsysbio/AI-MD_RNASeq_ChIPseq/blob/0f2ddcddc38f5d0eaa5b1712d467482b2d0a1ba5/STAR/
+maintainer: 'KoHaku55'
+description: Spliced Transcripts Alignment to a Reference.
+latest:
```

Comment:
This might be redundant but you’ll also need a tags section here (with latest) to update if there are ever other versions.

Reply:
Confirmed in CI because recipe did not validate. 

![03ADC780-3CEE-41BE-BB03-94E267251EC2](https://user-images.githubusercontent.com/814322/150511589-5ca22bfa-1bdf-4535-9c52-23e89d02c93a.jpeg)